issues.go

```diff
@@ -162,17 +163,30 @@ func issueSearch(ctx context.Context, label, token string, ch chan<- Issue) erro
 			CreatedAt time.Time `json:"created_at"`
 			URL       string    `json:"url"`
 			RepoURL   string    `json:"repository_url"`
+			Labels    []struct {
+				Name  string `json:"name"`
+				Color string `json:"color"`
+			} `json:"labels"`
 		} `json:"items"`
 	}
 	if err := json.NewDecoder(resp.Body).Decode(&data); err != nil {
 		return errors.Wrap(err, "could not decode json")
 	}
 
 	for _, item := range data.Items {
+		labels := make(map[string]string)
+		for _, label := range item.Labels {
+			// Show only labels that are not related to hacktoberfest.
+			if label.Name != "help wanted" && label.Name != "hacktoberfest" {
```

Comment:
The list of labels we're pulling is on line 50 and I'd rather not repeat them here. Maybe move the slice from line 50 up to the package scope around line 23 then this bit here can loop through that slice to check when making the map.

Reply:
Ah, I see that. Good idea! I'll get that changed tonight.